src/utils/tree_node.C

```diff
@@ -180,30 +180,33 @@ void TreeNode<N>::set_bounding_box (const std::pair<Point, Point>& bbox)
 
 
 template <unsigned int N>
-bool TreeNode<N>::bounds_node (const Node* nd) const
+bool TreeNode<N>::bounds_node (const Node* nd,
+                               Real relative_tol) const
 {
   libmesh_assert(nd);
-  return bounds_point(*nd);
+  return bounds_point(*nd, relative_tol);
 }
 
 
 
 template <unsigned int N>
-bool TreeNode<N>::bounds_point (const Point& p) const
+bool TreeNode<N>::bounds_point (const Point& p,
+                                Real relative_tol) const
 {
   const Point& min = bounding_box.first;
   const Point& max = bounding_box.second;
 
+  const Real tol = (max - min).size();
```

Comment:
Am I missing something?  You're passing in `relative_tol` here but not using it... this will currently lead to checking a lot more TreeNodes and hence elements (you're basically making the bounding boxes ~3x larger than they actually are).